mne/beamformer/_rap_music.py

```diff
@@ -172,9 +172,20 @@ def _make_dipoles(times, poss, oris, sol, gof):
 def _compute_subcorr(G, phi_sig):
     """Compute the subspace correlation."""
     Ug, Sg, Vg = linalg.svd(G, full_matrices=False)
+    # Now we look at the actual rank of the forward fields
+    # in G and handle the fact that it might be rank defficient
+    # eg. when using MEG and a sphere model for which the
+    # radial component will be truly 0.
+    # rank = np.sum(Sg > (Sg[0] * 1e-12))
```

Comment:
why have the same line commented out here?

Also, did you actually look at the eigenvalue spectra to see that it is safe to use `1e-12`? It would be good to visually inspect to double check